python/runtime/interop/PythonCppInterop.cpp

```diff
@@ -0,0 +1,100 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+#include "PythonCppInterop.h"
+#include "cudaq.h"
+
+namespace cudaq {
```

Comment:
I wonder if it might not be a good idea to put the python specific code in a sub-namespace like `cudaq::python` just to declutter things and prevent us from accidentally reusing the same names for C++ and Python both.

Reply:
good idea, fixed